Automattic/sensei
# Remove Sensei_WP_Cli class

Fixes #2800.

This PR removes instantiation of the `Sensei_WP_Cli` class, [which is incomplete](https://github.com/Automattic/sensei/blob/master/includes/class-sensei-wp-cli.php) at best. I'm wondering if we should remove the entire class file?

## Testing Instructions
Nothing much, but wouldn't hurt to click around the admin area and the front-end to ensure nothing is broken.
